### ISSUE TITLE
Inflatable Habitation Spaces cost/support seems off.  May need some rebalancing.

### ISSUE THREAD

**Issue by kkdredd** [BobPalmer/MKS#1081]
The Material Kit and Electric Charge cost to deploy a Ranger inflatable hab is the same as a Tundra 4.25 inflatable hab (8000MK/500EC).  However, the Ranger's BaseKerbalMonths of 16.8 is closer to the Tundra 2.75 Inflatable hab which has BaseKerbalMonths of 19.8 as opposed to the Tundra 4.75's BaseKerbalMonths of 77.  The Tundra 2.75 deploy cost is only 1000MK/100EC.  

The Ranger AG module has a deploy cost of 1000MK/100EC and the Ranger Workshop is 2000MK/200EC.  I would think that the Ranger Hab should be 1000MK/100EC since it is of comparable size to the Ranger AG module and roughly the same benefit as the Tundra 2.75.  However, I'm not sure of what was intended.

**Comment by llinard** [comment]
I would agree that it does seem rather "expensive" for an inflatable module particularly compared to the others that are available.

Seems like 1,000 - 2,000 would probably be more reasonable.
